sandbox/tests/test scenes/many light sampling/grid_EMT.py

```diff
@@ -0,0 +1,304 @@
+                                            "disney_material",
+                                            "01 - Default_mat",
+                                            {
+                                                "alpha_map": "1",
+                                                "layer1": {
+                                                    "anisotropic": "0",
+                                                    "base_color": "[1, 1, 1]",
+                                                    "clearcoat": "0",
+                                                    "clearcoat_gloss": "0",
+                                                    "layer_name": "layer1",
+                                                    "layer_number": "0",
+                                                    "mask": "1.0",
+                                                    "metallic": "0",
+                                                    "roughness": "1",
+                                                    "sheen": "0",
+                                                    "sheen_tint": "0",
+                                                    "specular": "0",
+                                                    "specular_tint": "0",
+                                                    "subsurface": "0.0"
+                                                }
+                                            }))
+
+    # Create light material.
+    assembly.materials().insert(asr.Material(
+                                            "generic_material",
+                                            "light_material",
+                                            {
+                                                "bump_amplitude": "1.0",
+                                                "displacement_method": "bump",
+                                                "edf": "light_material_edf",
+                                                "normal_map_up": "z",
+                                                "shade_alpha_cutouts": "false"
+                                            }))
+
+
+    #------------------------------------------------------------------------
+    # Geometry
+    #------------------------------------------------------------------------
+
+    # Load the scene geometry from disk.
+    objects = asr.MeshObjectReader.read(project.get_search_paths(), "plane", {"filename": "Plane001.binarymesh"})
+
+    # Insert all the objects into the assembly.
+    for object in objects:
+        # Create an instance of this object and insert it into the assembly.
+        instance_name = object.get_name() + "_inst"
+        material_name = {"material_slot_0": "01 - Default_mat"}
+        mat = orientation * asr.Matrix4d.make_translation(asr.Vector3d(0.0, 0.0, 0.0))
+        instance = asr.ObjectInstance(
+                                        instance_name,
+                                        {"visibility":
```

Comment:
Full visibility is the default, you don't need to specify anything in this case.